src/site/content/en/blog/contact-picker/index.md

```diff
@@ -0,0 +1,321 @@
+---
+title: A contact picker for the web
+subhead: The Contact Picker API provides an easy way for users to share contacts from their contact list.
+authors:
+  - petelepage
+description: Access to the user's contacts has been a feature of native apps since (almost) the dawn of time. The Contact Picker API is an on-demand API  that allows users to select an entry or entries from their contact list and share limited details of the selected contact(s) with a website. It allows users to share only what they want, when they want, and makes it easier for users to reach and connect with their friends and family.
+date: 2019-08-07
+updated: 2019-10-10
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - contacts
+  - chrome77
+hero: hero.jpg
+alt: Telephone on yellow background.
+draft: true
+---
+
+{% Aside %}
+  The Contact Picker API begins an origin trial in Chrome 77
+  (stable in September) as part of our capabilities project. We'll keep this
+  post updated as the implementation progresses.
+{% endAside %}
+
+## What is the Contact Picker API? {: #what }
+
+<style>
+  #video-demo { max-height: 600px; }
+</style>
+
+<a href="https://storage.googleapis.com/webfundamentals-assets/contact-picker/contact-picker.mp4">
+  <video id="video-demo" loop autoplay muted
```

Comment:
I think if you wrap this in a figure element instead of putting the w-figure classes on the video, then it will center properly on tablets and mobile.

Reply:
Will do